index.html

```diff
@@ -4848,6 +4855,29 @@ <h2 id="AudioWorkletProcessor">
                 This method is called synchronously by the audio <a>rendering
                 thread</a> that executes in an isochronous fashion.
               </p>
+              <p>
+                The return value of this method controls the lifetime of the
+                <a>AudioWorkletNode</a> corresponding to this instance of
+                <a>AudioWorkletProcessor</a>. It does so by determining the
+                existence of the node's <a href="#dfn-active-reference">active
+                reference</a>. At the conclusion of each call to the
+                <code>process()</code> method, if the return value is identical
+                to <code>true</code>, then an active reference is retained to
+                the node, affecting the node's <a href=
+                "#lifetime-AudioNode">lifetime</a>. If the value is other than
+                <code>true</code> (for example, <code>false</code> or
+                <code>undefined</code>), then any active reference is removed
+                and the node's lifetime depends only on other references as
+                they may exist.
```

Comment:
There is too much prose here talking about possible coercion of the value. Simply writing "is equal to true" should be enough, maybe referencing ES or something.